src/main/java/marquez/service/models/Datasource.java

```diff
@@ -0,0 +1,20 @@
+package marquez.service.models;
+
+import java.time.Instant;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import marquez.common.models.ConnectionUrl;
+import marquez.common.models.DatasourceName;
+
+@RequiredArgsConstructor
+@EqualsAndHashCode
+@ToString
+public class Datasource {
+
```

Comment:
minor: remove white space :)

Reply:
Will be fixed in next revision.

Reply:
Please see PRs #394, #387, and #384, where this is fixed.